libs/components/forms/testing/src/form-error/form-errors-harness.ts

```diff
@@ -35,84 +24,73 @@ export class SkyFormErrorsHarness extends SkyComponentHarness {
   }
 
   /*
-   * Gets total number of errors.
-   */
-  public async getNumberOfErrors(): Promise<number> {
-    return (await this.#getFormError()).length;
-  }
-
-  /*
-   * Gets if the required error has triggered.
+   * Whether the required field is set to an empty value.
    */
   public async hasRequiredError(): Promise<boolean> {
-    return (await this.#getFormErrorsClasses()).includes(
-      'sky-form-error-required',
-    );
+    return this.#hasErrorOfType('required');
   }
 
   /*
-   * Gets if the maximum length error has triggered.
+   * Whether the field has more characters than is allowed.
    */
   public async hasMaxLengthError(): Promise<boolean> {
-    return (await this.#getFormErrorsClasses()).includes(
-      'sky-form-error-maxlength',
-    );
+    return this.#hasErrorOfType('maxlength');
   }
 
   /*
-   * Gets if the minimum length has triggered.
+   * Whether the field has fewer characters than is allowed.
    */
   public async hasMinLengthError(): Promise<boolean> {
-    return (await this.#getFormErrorsClasses()).includes(
-      'sky-form-error-minlength',
-    );
+    return this.#hasErrorOfType('minlength');
   }
 
   /*
-   * Gets if the character count error has triggered.
+   * Whether the field has exceeded its maximum character count allowance.
```

Comment:
```suggestion
   * Whether the field has more characters than allowed.
```
Seems like we should use parallel construction in the wording of the JSDocs for the min/max entries, so I'd tweak this to follow the wording of the minLength item. If you prefer this wording, I'd reverse course and update the minLength item instead, although in that case, I'd tweak this item slightly first to change "has exceeded" to "exceeds" and delete "allowance."